install.js

```diff
@@ -30,7 +30,7 @@ const downloadHost = process.env.PUPPETEER_DOWNLOAD_HOST || process.env.npm_conf
 const puppeteer = require('./index');
 const browserFetcher = puppeteer.createBrowserFetcher({ host: downloadHost });
 
-const revision = require('./package.json').puppeteer.chromium_revision;
+const revision = process.env.PUPPETEER_CHROMIUM_REVISION || require('./package.json').puppeteer.chromium_revision;
```

Comment:
Note, we do have a `BrowserFetcher` API that serves this purpose:
https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-browserfetcher

But this one liner is a nicer than writing some code. @aslushnikov wdyt?

We'd need to also document this env var in https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#environment-variables.

Reply:
I've added the documentation as well FYI